.travis.yml

```diff
@@ -18,7 +18,6 @@ before_install:
 
 install:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install -y libao-dev libfftw3-dev librtlsdr-dev mingw-w64 python3-pyaudio; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo ln -s /usr/bin/ccache /usr/lib/ccache/i686-w64-mingw32-gcc; sudo ln -s /usr/bin/ccache /usr/lib/ccache/x86_64-w64-mingw32-gcc; fi
```

Comment:
It looks like this is no longer necessary on Ubuntu 18.04:
```
ln: failed to create symbolic link '/usr/lib/ccache/i686-w64-mingw32-gcc': File exists
ln: failed to create symbolic link '/usr/lib/ccache/x86_64-w64-mingw32-gcc': File exists
```
My second build was much faster than the first, so ccache appears to be working with mingw.